### PR TITLE
New package: ElementTechnologie.openElement version 1.57.9

### LINKED ISSUE #156096: [Package Request]: openElement
### How can we help?

I would like someone else to build the manifest.

### Please read and ensure the following

- [X] The installer meets the above requirements

### Please provide the following information

Download Page Url: https://openelement.uk/Download.htm
Publisher: [Element Technologie](https://dbpedia.org/page/OpenElement)
Package Name: openElement
Description: `WEBSITE EDITOR WITH TOTAL CONTROL - MODIFY ANY ELEMENT AND STYLE THE WAY YOU WANT`
Package Version: `1.57 R9`
Installer URL: https://openelement.uk/Files/Other/openElement.exe

### PR DESCRIPTION
**Pull request by Exorcism0666**
Checklist for Pull Requests
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?
- Resolve #156096

Manifests
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/156187)

### CODE REVIEW

**Comment by Exorcism0666** [comment]
## For moderators

This account is automated by Github Actions and the source code was created by CoolPlayLin. If you have any questions about any pull request, don't hesitate to ping Exorcism0666.
>[!warning]
>With the new version of Komac, numerous Pull Requests may contain bugs or unwanted elements. If such issues arise, please tag russellbanks so that these bugs can be addressed in future updates.

>[!important]
>Please carefully review these Pull Request before merging. If it is a Pull Request for removing incorrect content and the URLs are issue, free upon manual checking, please close this Pull Request directly. (It is best to inform Exorcism0666 of the closure.)

良い一日をお過ごしください！

**Comment by stephengillie** [comment]
HI @Exorcism0666,

This appears to be a nested installer - running the application opens an installer window: 
image

**Comment by Exorcism0666** [comment]
@Dragon1573 a little help wouldn't go amiss :)

**Comment by Dragon1573** [comment]
@Exorcism0666 - _Komac_ also regard this package as a portable executable.

Try manually run the installer file with `/?` switch, the following dialog shows.

image

**Comment by SpecterShell** [comment]
Advanced Installer

File: manifests/e/ElementTechnologie/openElement/1.57.9/ElementTechnologie.openElement.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: ElementTechnologie.openElement
+PackageVersion: 1.57.9
+InstallerType: portable
```
Comment (Dragon1573) [inline manifests/e/ElementTechnologie/openElement/1.57.9/ElementTechnologie.openElement.installer.yaml:6]:
```suggestion
InstallerType: exe
```

This is an installer wizard created by _Advanced Installer_. It's not a portable executable.

File: manifests/e/ElementTechnologie/openElement/1.57.9/ElementTechnologie.openElement.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: ElementTechnologie.openElement
+PackageVersion: 1.57.9
+InstallerType: portable
+Installers:
+- Architecture: x86
+  InstallerUrl: https://openelement.uk/Files/Other/openElement.exe
+  InstallerSha256: 22F33F86128B59D3911702150EDA6438C2275065C765EE550491A601FF568AE6
```
Comment (Dragon1573) [inline manifests/e/ElementTechnologie/openElement/1.57.9/ElementTechnologie.openElement.installer.yaml:10]:
```suggestion
  InstallerSha256: 22F33F86128B59D3911702150EDA6438C2275065C765EE550491A601FF568AE6
  InstallerLocale: en-US
  InstallerSwitches:
    Custom: /exelang 1033 /exenoui /qn
    InstallLocation: '/extract:<INSTALLPATH>'
    Log: /exelog <LOGPATH>
```

- This package contains both English (language ID `1033`) and French (language ID `1036`) profile.
- `InstallerSwitches` can be found by adding `/?` switch to the installer file.

**Review by Dragon1573 — CHANGES_REQUESTED** [review]
Suggestions were tested in `amd64` Windows Sandbox instance.

**Review by Dragon1573 — APPROVED** [review]
Verified in _Windows Sandbox_.

More tests should be performed by moderators.

**Comment by stephengillie** [comment]
Verify: 
image
- I might have lost Internet (hotspot hiccup) during the install process - would that result in this error? Is it a local page that's downloaded durng install? Because it persists despite having a solid network connection.

image

**Comment by Dragon1573** [comment]
This can be reproduced on my _Windows Sandbox_ instance.

For some well-known reason, I'm behind a proxy. Even though my proxy is enabled and valid, I cant' load that page either.

On the screen it says "Internet connection is not required", so I think this situation is acceptable.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**